Kickstarter-iOS/DataSources/SearchDataSource.swift

```diff
@@ -32,7 +32,9 @@ internal final class SearchDataSource: ValueCellDataSource {
 
     if !projects.isEmpty {
       projects.suffix(from: 1).forEach {
```

Comment:
I believe this could also be written as:

`projects.dropFirst().forEach { ... }`, the `Sequence` type gives us a `dropFirst` function which returns everything but the first element.